hub/core/chunk_engine/write.py

```diff
@@ -14,11 +15,19 @@
 from hub.util.array import normalize_and_batchify_shape
 
 
+def write_tensor_meta(key: str, storage: StorageProvider, meta: dict):
+    storage[get_meta_key(key)] = pickle.dumps(meta)
```

Comment:
I'd personally prefer a human-readable meta file like a json instead of a pickle.

Reply:
Me too, I don't know why we are doing it this way. @McCrearyD had some TODO comments about removing the use of pickle, I'm just following suit and factoring out